next.config.js

```diff
@@ -0,0 +1,76 @@
+module.exports = {
+  webpack: (config, { buildId, dev, isServer, defaultLoaders, webpack }) => {
+    const entry = config.entry;
+    config.entry = () => {
+      return entry().then(e => ({
+        ...e,
+        parserWorker: "./src/devtools/client/debugger/src/workers/parser/worker",
+        searchWorker: "./src/devtools/client/debugger/src/workers/search/worker",
+      }));
+    };
+
+    // handles build error from webpack/runtime/compat
+    if (isServer) {
```

Comment:
not sure if we need this if we're relying on vercel to build

Reply:
I don't know if it's required. I'm following a reference in the (now linked) issue that said they used that to avoid extra size in their client bundle.